setup.py

```diff
@@ -11,7 +11,7 @@
 tests_require = ['WebTest']
 
 setup(name='pyramid_nacl_session',
-      version='0.4.dev0',
+      version='0.4',
```

Comment:
Don't drop the `.dev0`, the person who does the release will modify this value when the time comes.